commit 97f0b7b25474fab25f5757f7c50a77e20be5d05b
Author: František Dvořák <valtri@civ.zcu.cz>
Date:   Mon Sep 7 15:00:48 2015 +0200

    Use the API properly in the example (format security error fixed).

diff --git a/examples/helloworld1/mylog.h b/examples/helloworld1/mylog.h
index 280f3dc..57e5369 100644
--- a/examples/helloworld1/mylog.h
+++ b/examples/helloworld1/mylog.h
@@ -39,7 +39,7 @@ static LOG4C_INLINE int mylog_fini(){
 
 static LOG4C_INLINE void mylog_msg(char *catName,int a_priority, char *msg){
 #ifndef WITHOUT_LOG4C
-	log4c_category_log(log4c_category_get(catName), a_priority, msg);
+	log4c_category_log(log4c_category_get(catName), a_priority, "%s", msg);
 #else
 	printf(msg);
 #endif
